packages/app-page-builder/src/templateEditor/Editor.tsx

```diff
@@ -108,7 +98,10 @@ export const TemplateEditor = () => {
                 const { content, ...restOfTemplateData } = pageTemplateData;
 
                 const existingContent = content
-                    ? { ...content, elements: getBlocksWithUniqueElementIds(content.elements) }
+                    ? {
+                          ...content,
+                          elements: content.elements
+                      }
```

Comment:
```suggestion
                    ? content
```

P.S. Not sure if we care about making a shallow clone here.

Reply:
Actually this can go away completely. We no longer need the `existingContent` variable.